src/schema/me/recently_viewed_artworks.ts

```diff
@@ -21,3 +26,31 @@ export const RecentlyViewedArtworks = {
     })
   },
 }
+
+export const recordArtworkViewMutation = mutationWithClientMutationId({
+  name: "RecordArtworkView",
+  description: "Records an artwork view.",
```

Comment:
Since this property is here, we could use it to specify what qualifies as a "view" (e.g., the work is the focus of a full browser page or occupies the majority of a viewport).

Reply:
Sure!